src/web/utils/severity.ts

```diff
@@ -220,3 +220,20 @@ export const getSeverityLevels = (
 export const getSeverityLevelBoundaries = (
   rating: SeverityRating = DEFAULT_SEVERITY_RATING,
 ): SeverityLevelBoundaries => SEVERITY_LEVEL_BOUNDARIES[rating];
+
+export const printPercentile = (percentile: number): string => {
+  if (isNumber(percentile)) {
+    if (
+      parseInt(percentile.toFixed(0)) % 10 > 3 ||
+      parseInt(percentile.toFixed(0)) % 10 === 0
+    )
+      return `${percentile.toFixed(0)}th`;
+    if (parseInt(percentile.toFixed(0)) % 10 === 1)
+      return `${percentile.toFixed(0)}st`;
+    if (parseInt(percentile.toFixed(0)) % 10 === 2)
+      return `${percentile.toFixed(0)}nd`;
+    if (parseInt(percentile.toFixed(0)) % 10 === 3)
+      return `${percentile.toFixed(0)}rd`;
+  }
+  return 'N/A';
+};
```

Comment:
I am also not sure if `parseInt` is necessary here.
```suggestion
export const renderPercentile = (percentile: number): string => {
  if (isNumber(percentile)) {
    const value = percentile.toFixed(0)
    const rest = parseInt(value) % 10;
    if (
       value > 3 ||
      value === 0
    ) {
      return `${value}th`;
    }
    if (rest === 1) {
      return `${value}st`;
    }
    if (rest === 2) {
      return `${value}nd`; 
    }
    if (rest === 3) {
      return `${value}rd`;
    }
  }
  return 'N/A';
};
```